src/qml/ElevationProfile.qml

```diff
@@ -29,14 +27,8 @@ Item {
     elevationProfileCanvas.clear();
   }
 
-  Rectangle {
-      id: backgroundRect
-
-      width: elevationProfile.width
-      height: elevationProfile.height
-      color: "#ffffff"
-      radius: 8
-  }
+  color: "#bbfafafa"
```

Comment:
shouldn't  we use Theme.mainBackgroundColorSemiOpaque as background color? 

Result of using Theme.mainBackgroundColorSemiOpaque:   

![Screenshot from 2024-05-20 17-36-21](https://github.com/opengisch/QField/assets/36326627/da104713-8eda-4f97-8cf6-80f6908afd7e)


Then i think `instrutionLabel` style should change.

Reply:
@mohsenD98 , we have to hard code it for now because the elevation plotting canvas doesn't expose a property to set the label color yet (and therefore the result is bad on dark theme ATM)